### DISCUSSION TITLE
Unable to build docs locally

### DISCUSSION THREAD

**Discussion by daniellepintz** [Lightning-AI/pytorch-lightning#12244] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I am trying to build docs locally. I followed the steps [here](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md#documentation) but when I go to `http://docs/build/html/index.html` I get: "This site can’t be reached"

Here is the end of the output for when I run `make html`:
```
copying images... [ 97%] _static/images/mnist_imgs/restart_runtime.png
copying images... [ 98%] _static/images/mnist_imgs/tpu_start.png
copying images... [ 99%] _static/images/mnist_imgs/tpu_fast.png
copying images... [100%] _static/images/general/PTL101_youtube_thumbnail.jpg

copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 1 warning.

The HTML pages are in build/html.
The name of the builder is: htmlCopying sphinx_paramlinks stylesheet... done
```

It looks all fine to me...

**Comment by akihironitta** [comment, marked as answer]
Hey @daniellepintz I think the URL should be something different. (the protocol shouldn't be http.)

In my env, the project root dir is:
```
/Users/nitta/work/github.com/PyTorchLightning/pytorch-lightning/
```
and the URL is:
```
file:///Users/nitta/work/github.com/PyTorchLightning/pytorch-lightning/docs/build/html/index.html
```

**Reply by daniellepintz** [reply]
thanks so much @akihironitta! I tried
`file:///home/daniellepintz/pytorch-lightning/docs/build/html/index.html` but unfortunately that still doesn't work for me. Although I am getting a different error now: "Your file couldn’t be accessed. It may have been moved, edited, or deleted."

No worries though, I was able to find a workaround by looking in the CI docs job
